test/tests/materials/piecewise_by_block_material/test.i

```diff
@@ -70,7 +70,7 @@
 [Postprocessors]
   # to trigger on boundary element computations
   [flux]
-    type = ADSideDiffusiveFluxIntegral
+    type = ADNonFunctorSideDiffusiveFluxIntegral
```

Comment:
I would just regold and delete the test object, it's just this test

Reply:
Using the production object, post-this-PR, would negate the whole purpose of this test which is to test piecewise-by-block restriction of traditional materials with finite volume objects

Reply:
fair point